doc/classes/SkeletonIK.xml

```diff
@@ -26,12 +28,14 @@
 			<argument index="0" name="one_time" type="bool" default="false">
 			</argument>
 			<description>
+                Starts applying IK effects on each frame to the [Skeleton] bones but will only take effect starting on the next frame. If used with a bool [code]one_time[/code] parameter will take effect immediately but also reset on next frame.
 			</description>
 		</method>
 		<method name="stop">
 			<return type="void">
 			</return>
 			<description>
+                Stops applying IK effects on each frame to the [Skeleton] bones but does not reset already applied bones positions. For a manual bone reset use the [Skeleton] clear_bones_global_pose_override() function.
```

Comment:
```suggestion
                Stops applying IK effects on each frame to the [Skeleton] bones but does not reset already applied bones positions. For a manual bone reset, use the [method Skeleton.clear_bones_global_pose_override] method.
```

Reply:
Is this intended or a typo since Skelton3D does not exist in Godot 3.x?